packages/ckeditor5-language/src/textfragmentlanguageediting.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module language/textfragmentlanguageediting
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+import TextFragmentLanguageCommand from './textfragmentlanguagecommand';
+import { stringifyLanguageAttribute, parseLanguageAttribute } from './utils';
+
+const ATTRIBUTE_KEY = 'language';
```

Comment:
This is the same as in the command, maybe we should export it so it could be imported in the command?

Reply:
As we talked, I've replaced it with plain string as in most cases we are not using const for attributes.